src/docker-osx-dev

```diff
@@ -409,7 +409,12 @@ function configure_docker_machine {
   DOCKER_MACHINE_DRIVER_NAME=$(inspect_docker_machine "{{.DriverName}}")
 
   DOCKER_HOST_SSH_URL="$DOCKER_HOST_USER@$DOCKER_HOST_IP"
-  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/id_rsa"
+  DOCKER_HOST_SSH_KEY="$DOCKER_MACHINE_STORE_PATH/machines/$DOCKER_MACHINE_NAME/id_rsa"
+
+  if [[ ! -f $DOCKER_HOST_SSH_KEY ]]; then
```

Comment:
I think this fix appears to fix two alternative situations that arise in docker-machine 0.5.0. For earlier versions of docker-machine, the file check test will always fail. I wonder if this is a bug in docker-machine, and goes back to the issue that all these fields in `inspect` are very poorly documented.